src/main/kotlin/com/devfloor/untitled/board/domain/Board.kt

```diff
@@ -39,4 +39,6 @@ class Board(
     @Enumerated(value = EnumType.STRING)
     @Column(name = "board_type")
     val type: BoardType = type
+
+    fun matchsType(type: BoardType): Boolean = this.type == type
```

Comment:
별건 아니지만 여기도 딴데처럼 matches로 맞춰주심 될거같아요 !
```suggestion
    fun matchesType(type: BoardType): Boolean = this.type == type
```